grizli/prep.py

```diff
@@ -430,7 +464,7 @@ def apply_region_mask(flt_file, dq_value=1024, verbose=True):
     """
     import pyregion
 
-    mask_files = glob.glob(flt_file.replace('_flt.fits', '.*.mask.reg').replace('_flc.fits', '.*.mask.reg').replace('_c0m.fits', '.*.mask.reg').replace('_c0f.fits', '.*.mask.reg'))
+    mask_files = glob.glob(flt_file.replace('_flt.fits', '.*.mask.reg').replace('_flc.fits', '.*.mask.reg').replace('_c0m.fits', '.*.mask.reg').replace('_c0f.fits', '.*.mask.reg').replace('_rate.fits','.*.mask.reg'))
```

Comment:
I'm embarrassed by this line.  Let's do the following to strip out any extension beyond the last `_` character.

```python
# e.g., iabc01defq_flt.fits > iabc01defq.*.mask.reg
mask_files = glob.glob('_'.join(flt_file.split('_')[:-1]) + '.*.mask.reg')
```

Reply:
Done here: 56af2ae77b750072824b7a7dcf92c47b43d3d4bd